python/tests/sync_tests/conftest.py

```diff
@@ -0,0 +1,157 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+from typing import Generator, List, Optional
+
+import pytest
+from glide.config import (
+    BackoffStrategy,
+    GlideClientConfiguration,
+    GlideClusterClientConfiguration,
+    NodeAddress,
+    ProtocolVersion,
+    ReadFrom,
+    ServerCredentials,
+)
+from glide.exceptions import ClosingError
+from glide.glide_client import GlideClient, GlideClusterClient
```

Comment:
why is it needed? it should be separated - and only the sync client to be used here